internal/cluster/cluster_test.go

```diff
@@ -1204,17 +1206,22 @@ func nonDefaultInventory() string {
 
 var _ = Describe("PrepareForInstallation", func() {
 	var (
-		ctx       = context.Background()
-		capi      API
-		db        *gorm.DB
-		clusterId strfmt.UUID
-		dbName    = "cluster_prepare_for_installation"
+		ctx          = context.Background()
+		capi         API
+		db           *gorm.DB
+		clusterId    strfmt.UUID
+		dbName       = "cluster_prepare_for_installation"
+		ctrl         *gomock.Controller
+		manifestsApi *mocks.MockManifestsAPI
 	)
 
 	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		manifestsApi = mocks.NewMockManifestsAPI(ctrl)
+		manifestsApi.EXPECT().CreateClusterManifest(gomock.Any(), gomock.Any()).Return(operations.NewCreateClusterManifestCreated()).AnyTimes()
```

Comment:
The reason i suggested to make `AddChronyManifest` into an interface is because of the tests as well. here you are aware of an implementation of a different package that will call to this api and you don't check a bad flow because if you do fail this function you will be actually testing the `network` package.